inbox/react/customization.mdx

```diff
@@ -1,205 +1,45 @@
 ---
-title: "Customizing Inbox"
-sidebarTitle: "Customization"
-description: "Learn how to customize the pre built Inbox component using the appearance props"
+title: 'Customizing Inbox'
+sidebarTitle: 'Customization'
+description: 'Learn how to customize the pre built Inbox component using the appearance props'
 ---
 
 ## Appearance Prop
 
-`appearance` prop can be used to customise the Inbox. It has three main keys: `baseTheme`, `variables` and `elements`.
+The `appearance` prop can be used to customise the Inbox. It has three main keys: `baseTheme`, `variables` and `elements`.
 
-- `variables`: The variables object allows you to set global styling variables that apply to multiple elements within the inbox. This can include colors, border radius, and other common styles. You can find the list of supported variables [below](#list-of-available-variables).
-- `elements`: The elements object allows you to define styles for specific elements within the inbox. Each key corresponds to an element, and the value is an object containing `style properties` for that element or you can also pass your `css classes`.
-  See list of available elements [below](#list-of-available-elements).
+- `variables`: The variables object allows you to set global styling variables that apply to multiple elements within the inbox. This can include colors, border radius, and other common styles. You can find the list of [supported variables](https://github.com/novuhq/novu/blob/01268c688d1c173736b4ad51147370844952335a/packages/js/src/ui/types.ts#L24).
+- `elements`: Elements are the individual UI components that make up the Inbox. The elements object allows you to define styles for these components. Each key corresponds to an component, and the value is an object containing `style properties` or you can also pass your `css classes`.
+  See list of available elements [here](https://github.com/novuhq/novu/blob/01268c688d1c173736b4ad51147370844952335a/packages/js/src/ui/config/appearanceKeys.ts#L6).
 - `baseTheme`: The baseTheme object allows you to define the base theme for the inbox. This can include variables and elements that apply to the entire inbox, as well as additional themes that can be switched between. It accepts an array of objects, each containing variables and elements. Each object will override the previous object's styles.
   This enables to override the custom themes imported from the `@novu/react` package.
-  Currently, we only export the `dark` theme.
+  Currently, we only export the `dark` [theme](https://github.com/novuhq/novu/blob/01268c688d1c173736b4ad51147370844952335a/packages/js/src/ui/config/defaultVariables.ts#L3).
 
-```jsx
+```tsx
 import { Inbox } from '@novu/react';
 const appearance = {
-          variables: {
```

Comment:
This code might become quickly outdated, so I provided a small amount to show what is possible. The appearance prop section has the link to the code where they can check all possible properties.